runtime/layers/fpm/Dockerfile

```diff
@@ -1,10 +1,13 @@
 ARG PHP_VERSION
 FROM bref/tmp/cleaned-build-php-$PHP_VERSION
+ARG PHP_VERSION
```

Comment:
Duplicate `ARG PHP_VERSION`?

Reply:
no, you need to re-declare `PHP_VERSION`. Every `FROM` in your Dockerfile creates a new build stage, which causes values of any ARG from previous stages to be lost.
https://benkyriakou.com/posts/docker-args-empty
I also didn't know this and it took me some time to figure this out.

Reply:
Okay, did not know that 😄

Reply:
Should we include a comment about that and link to that post to not confuse others?

Reply:
IMO it's very Docker related, probably even very basic for someone that has more experience with Docker. Also, I think that it's why git/github is for: so people can trackdown changes and check why they were made (e.g.: git blame, history, etc..)

Reply:
I'm always up for comments in the code but I agree too that it doesn't seem necessary to me here.